src/Files.App/ViewModels/UserControls/Previews/ShellPreviewViewModel.cs

```diff
@@ -114,23 +115,23 @@ public void LoadPreview(UIElement presenter)
 
 		private bool ChildWindowToXaml(IntPtr parent, UIElement presenter)
 		{
-			D3D_DRIVER_TYPE[] driverTypes =
+			DirectN.D3D_DRIVER_TYPE[] driverTypes =
```

Comment:
Change should be unnecessary after merging main

Reply:
Done